greenplumpython/table.py

```diff
@@ -509,6 +510,16 @@ def explain(self, format: str = "TEXT") -> Iterable:
         assert results is not None
         return results
 
+    # FIXME : define func type
+    # FIXME : Add more tests
+    def apply(self, func) -> "FunctionCall":
+        LAMBDA = lambda: 0
+        if isinstance(func, type(LAMBDA)) and func.__name__ == LAMBDA.__name__:
+            return func(self)
```

Comment:
I think this lines are making sure this func is a lambda func?
maybe this is a little better?
```python
if callable(func) and func.__name__ == (lambda:_).__name__:
    return func(self)
```

Reply:
How to better resolve this report error?
` error: "_" is not defined (reportUndefinedVariable)`

Reply:
Will change it to `if callable(func) and func.__name__ == "<lambda>":`